### PR TITLE
Include superceded by information

### PR DESCRIPTION
**Pull request by jonescc**

### CODE REVIEW

File: api/src/main/java/au/org/aodn/nrmn/restapi/dto/species/SpeciesDto.java [hunk 1]
```diff
@@ -21,4 +23,10 @@
 
     private String order;
 
+    private String rank;
+
+    private Boolean superceded;
+
+    private String validName;
```
Comment (nspool) [inline api/src/main/java/au/org/aodn/nrmn/restapi/dto/species/SpeciesDto.java:30]: Is `validName` the name of the species which has superseded this one?

Comment (jonescc) [reply api/src/main/java/au/org/aodn/nrmn/restapi/dto/species/SpeciesDto.java:30]: Yes

Comment (jonescc) [reply api/src/main/java/au/org/aodn/nrmn/restapi/dto/species/SpeciesDto.java:30]: I can include the aphiaId for that as well.  I'll just add that now

Comment (jonescc) [reply api/src/main/java/au/org/aodn/nrmn/restapi/dto/species/SpeciesDto.java]: Actually, just managed to get a hold of @bpasquer  and she only wants superseded by added as a column after species name.  Removing displayName and rank.

Comment (jonescc) [reply api/src/main/java/au/org/aodn/nrmn/restapi/dto/species/SpeciesDto.java]: Otherwise existing functionality is OK

**Review by nspool — APPROVED** [review]

**Pull request merged**